components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -2451,6 +2465,20 @@ private void parseRefreshTokenRenewalConfiguration(OMElement oauthConfigElem) {
         }
     }
 
+    private void parseRefreshTokenGrantValidationConfiguration(OMElement oauthConfigElem) {
```

Comment:
is this method name misleading? better have authenticated user term with it

Reply:
My thought was that there could be future refresh grant validation configurations beyond just the authenticated user. In that case, we could reuse this configuration method and read those settings through a single method. Does that make sense?

Reply:
+1